VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -228,6 +233,23 @@ public String getNewRelicKey() {
     }
 
     /**
+     * End to end test configurations
```

Comment:
'End to end' should be hyphenated, i.e. 'End-to-end'. Similarly in other comments as well.